main.py

```diff
@@ -160,94 +162,97 @@ def is_ignored(field, ignored_range):
-            stored_flow_obj = serializer.from_dict(stored_flow["flow"], switch)
-
-            flow = {"flows": [stored_flow["flow"]]}
+        for cookie, stored_flows in self.stored_flows[dpid].items():
+            for stored_flow in stored_flows:
+                stored_time = get_time(
+                    stored_flow.get("created_at", "0001-01-01T00:00:00")
+                )
+                if (now() - stored_time).seconds <= STATS_INTERVAL:
+                    continue
+                stored_flow_obj = serializer.from_dict(stored_flow["flow"], switch)
+                if stored_flow_obj in installed_flows[cookie]:
+                    continue
 
-            if stored_flow_obj not in switch.flows:
-                if command == "add":
-                    log.info("A consistency problem was detected in " f"switch {dpid}.")
-                    self._install_flows(command, flow, [switch], save=False)
-                    log.info(
-                        f"Flow forwarded to switch {dpid} to be "
-                        f"installed. Flow: {flow}"
-                    )
+                log.info(f"Consistency check: missing flow on switch {dpid}.")
+                flow = {"flows": [stored_flow["flow"]]}
+                self._install_flows("add", flow, [switch], save=False)
+                log.info(
+                    f"Flow forwarded to switch {dpid} to be installed. Flow: {flow}"
+                )
 
     def check_storehouse_consistency(self, switch):
         """Check consistency of installed flows for a specific switch."""
         dpid = switch.dpid
 
-        for installed_flow in switch.flows:
-
-            # Check if the flow is in the ignored flow list
-            if self.consistency_ignored_check(installed_flow):
+        for cookie, flows in self.switch_flows_by_cookie(switch).items():
+            if self.is_ignored(cookie, self.cookie_ignored_range):
                 continue
 
-            if dpid not in self.stored_flows:
-                log.info("A consistency problem was detected in " f"switch {dpid}.")
-                flow = {"flows": [installed_flow.as_dict()]}
-                command = "delete_strict"
-                self._install_flows(command, flow, [switch], save=False)
-                log.info(
-                    f"Flow forwarded to switch {dpid} to be deleted." f" Flow: {flow}"
-                )
-            else:
-                serializer = FlowFactory.get_class(switch)
-                stored_flows = self.stored_flows[dpid]["flow_list"]
-                stored_flows_list = [
-                    serializer.from_dict(stored_flow["flow"], switch)
-                    for stored_flow in stored_flows
-                ]
+            serializer = FlowFactory.get_class(switch)
+            stored_flows_list = [
+                serializer.from_dict(stored_flow["flow"], switch)
+                for stored_flow in self.stored_flows[dpid].get(cookie, [])
+            ]
+            log.debug(
```

Comment:
Added this `log.debug` that can be a bit verbose if we have a lot of flows stored, but without this it would be really hard to debug if in the future we end with alien flows being accidentally deleted. The archived flows and other better messages will be address on this issue https://github.com/kytos-ng/flow_manager/issues/33